supply-chain.md

```diff
@@ -8,196 +8,170 @@ nav_order: 1
 ```bash
 bin/guacone collect files ../guac-data/docs/
 ```
 
-This will ingest a collection of SBOMs and SLSA attestations into GUAC.
-
 Once ingested we will see the following message (the number of documents may
 vary):
 
 ```bash
 {"level":"info","ts":1681864775.1161852,"caller":"cmd/files.go:201","msg":"completed ingesting 67 documents of 67"}
 ```
 
-### Setting up the experimental GUAC Visualizer
+## Step 4: Set up the experimental GUAC Visualizer
 
-To start up the GUAC visualizer, follow
-[this guide](https://github.com/guacsec/guac-visualizer/tree/main/docs/setup.md).
+[Start up the GUAC visualizer](https://guac.sh/guac-visualizer/).
 
-## New security incident
+## Step 5: Mark packages as bad when a security incident occurs
 
-A new security incident has occurred and listening on various communities, we
-know that a particular package is affected, more specifically the package
-"maven/org.apache.logging.log4j/log4j-core" has been found to have a
-vulnerability (yikes!).
+A new security incident has occurred and various communities have pointed out that a particular package is affected. In this scenario, the package "maven/org.apache.logging.log4j/log4j-core" has a vulnerability (yikes!).
 
-The first step we can take is to mark this package as bad. We can do this by
-using the `guacone certify` command which defaults to assert a negative
-certification (instead of a positive one), as well as a `justification` to
-indicate why it is bad. In this case, it is a critical vulnerability.
+The first step we can take is to mark this package as bad by using the `guacone certify` command. This command defaults to assert a negative
+certification (instead of a positive one), as well as a `justification` to indicate why the package is bad. In this case, it is a critical vulnerability:
 
 ```bash
 bin/guacone certify package "never use this version of log4j" "pkg:maven/org.apache.logging.log4j/log4j-core@2.8.1"
 ```
 
-an output will contain the following meaning that we have successfully added
-"CertifyBad":
+If we successfully added "CertifyBad", the output will show:
 
 ```bash
 {"level":"info","ts":1683130083.9894989,"caller":"helpers/assembler.go:69","msg":"assembling CertifyBad: 1"}
 ```
 
-## Are you affected?
+## Step 6: Check if you are affected
 
-To answer the question of "Are you affected by this?" or "What do I need to
-patch", you can utilize one of our experimental GUAC tools, the guac-visualizer.
-The GUAC visualizer provides a utility to do some basic analysis and exploration
-of the software supply chain, which is a great way to get a sense of the size of
-the problem and to help in developing prototype utilities and queries with GUAC
-(very much like the [vulnerability CLI](/demo/query_vuln.md)).
+To find out if you're affected by the security incident and decide what you need to patch, utilize the [Guac Visualizer](https://guac.sh/guac-visualizer/). The GUAC visualizer provides a utility to do some basic analysis and exploration
```

Comment:
Is Step 6 needed? Step 4 covers running the visualizer. I'm not sure why we have this here as well.

Reply:
Yes, I agree. We can combine step 4 and 6 together